docs/sphinx_api_docs_source/build_sphinx_api_docs.py

```diff
@@ -208,11 +208,16 @@ def _build_class_md_stubs(self) -> None:
         for definition in definitions:
             if isinstance(definition.ast_definition, ast.ClassDef):
                 md_stub = self._create_class_md_stub(definition=definition)
-                self._write_stub(stub=md_stub, definition=definition)
+                self._write_stub(
+                    stub=md_stub,
+                    file_name=self._get_md_file_name_from_entity_name(
```

Comment:
The changes in this file are to make sure that the class name is used for classes and the module name (e.g. `great_expectations.util`) is used for modules with module level functions that are part of the public API (e.g. `get_context()`)

Reply:
Thank you 🙇🏼 that sounds great